Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -77,6 +78,7 @@ REFLECT_NODE_BEGIN( ObjectNode, Node, MetaNone() )
     REFLECT( m_AllowDistribution,                   "AllowDistribution",                MetaOptional() )
     REFLECT( m_AllowCaching,                        "AllowCaching",                     MetaOptional() )
     REFLECT_ARRAY( m_CompilerForceUsing,            "CompilerForceUsing",               MetaOptional() + MetaFile() )
+    REFLECT( m_ClangTidyConfigurationFile,          "ClangTidyConfigurationFile",       MetaOptional() + MetaFile() )
```

Comment:
This will add a string to every object file, whether using clang tidy or not.

I wonder if there's a reasonable way to avoid this.